NuRadioMC/SignalGen/ARZ/ARZ.py

```diff
@@ -483,10 +515,6 @@ def get_time_trace(self, shower_energy, theta, N, dt, shower_type, n_index, R, s
             number of samples in the time domain
         dt: float
             size of one time bin in units of time
-        profile_depth: array of floats
```

Comment:
This PR also fixes these doc-strings which refer to arguments that do not exist.